docs/user-guide/src/bmo/troubleshooting.md

```diff
@@ -0,0 +1,30 @@
+# Troubleshooting FAQ
+
+## Host is stuck in cleaning, how do I delete it?
+
+First and foremost, avoid using forced deletion, otherwise you'll have [a
+conflict](#mac-address-conflict-on-registration). If you don't care about disks
+being [cleaned](automated_cleaning.md), you can edit the BareMetalHost resource
+and disable cleaning:
+
+```yaml
+spec:
+  automatedCleaningMode: disabled
+```
+
+Alternatively, you can wait for 3 cleaning retries to finish. After that, the
+host will be deleted. If you do care about cleaning, you need to figure out why
+it does not finish.
+
+## MAC address conflict on registration
+
+If you force deletion of a host after registration, BareMetal Operator will not
+be able to delete the corresponding record from Ironic. If you try to enroll
+the same host again, you will see the following error:
+
+`Normal  RegistrationError  4m36s  metal3-baremetal-controller  MAC address 11:22:33:44:55:66 conflicts with existing node namespace~name`
```

Comment:
Use three ticks instead of one as it is a paragraph of its own and not embedded in sentence.

Reply:
The reason I haven't done it is:

docs/user-guide/src/bmo/troubleshooting.md:25 MD040/fenced-code-language Fenced code blocks should have a language specified [Context: "```"]

Reply:
It complains if you have  ` '''console` (replace '''' with actual ticks) ?

Reply:
Maybe just '''text block then, if you feel console is wrong? I think we have quite a lot of plain output marked as console?